powerauth-webauth-authentication/src/main/java/io/getlime/security/powerauth/lib/webauth/authentication/configuration/WebAuthServicesConfiguration.java

```diff
@@ -57,7 +81,44 @@ public CredentialStoreClient defaultCredentialStoreClient() {
      */
     @Bean
     public NextStepClient defaultNextStepClient() {
-        return new NextStepClient(nextstepServiceUrl);
+        NextStepClient client = new NextStepClient(nextstepServiceUrl);
+        // whether invalid SSL certificates should be accepted
+        if (acceptInvalidSslCertificateInNextStep) {
+            trustAllCertificates();
+        }
+        return client;
     }
 
+    /**
+     * Activate trust in all SSL certificates including invalid ones for non-production use.
+     */
+    private void trustAllCertificates() {
```

Comment:
You are right. I think it is enough to have one property, this setting is used for development as far as I understand.